tools/common/MachO.cs

```diff
@@ -133,8 +134,18 @@ public enum LoadCommands : uint
 			MintvOS = 0x2f,//#define LC_VERSION_MIN_TVOS 0x2F /* build for AppleTV min OS version */
 			MinwatchOS = 0x30,//#define LC_VERSION_MIN_WATCHOS 0x30 /* build for Watch min OS version */
 			//#define LC_NOTE 0x31 /* arbitrary data included within a Mach-O file */
-			//#define LC_BUILD_VERSION 0x32 /* build for platform min OS version */
+			BuildVersion = 0x32,//#define LC_BUILD_VERSION 0x32 /* build for platform min OS version */
+		}
 
+		public enum Platform : uint {
+			PLATFORM_MACOS = 1,
+			PLATFORM_IOS = 2,
+			PLATFORM_TVOS = 3,
+			PLATFORM_WATCHOS = 4,
+			PLATFORM_BRIDGEOS = 5,
+			PLATFORM_IOSSIMULATOR = 7,
+			PLATFORM_TVOSSIMULATOR = 8,
+			PLATFORM_WATCHOSSIMULATOR = 9,
```

Comment:
minor (we don't ship it) but the `PLATFORM*` prefix and casing is not the one we normally use.